source/MaterialXView/Viewer.cpp

```diff
@@ -868,6 +880,8 @@ void Viewer::createAdvancedSettings(Widget* parent)
     referenceQualityBox->setCallback([this](bool enable)
     {
         _genContext.getOptions().hwDirectionalAlbedoMethod = enable ? mx::DIRECTIONAL_ALBEDO_MONTE_CARLO : mx::DIRECTIONAL_ALBEDO_TABLE;
+        // No Albedo Table support for Essl yet.
```

Comment:
For ESSL code, I'd recommend always rendering with DIRECTIONAL_ALBEDO_CURVE_FIT, since it's the most performant of the three options.  Monte Carlo sampling for directional albedo would likely be too heavyweight in the contexts where ESSL code is run.

Reply:
This was a copy and paste mistake. The initialization is to CURVE FIT so this will always use curve fit as table support does not exist currently.